<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFlowPhase</class>
 <widget class="QDialog" name="DialogFlowPhase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>跟随相位表</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>17</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>跟随相位表号</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>581</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="title">
    <string>操作类型</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>28</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>空</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioother">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>28</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>其他</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radionormal">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>28</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>正常</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiomin">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>26</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>最小绿灯黄灯</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="title">
    <string>包含相位</string>
   </property>
   <widget class="QCheckBox" name="bao">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位1 </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>48</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位11</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_12">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位12</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_13">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位13</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_14">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位14</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_15">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位15</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bao_16">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>48</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位16</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="title">
    <string>修正相位</string>
   </property>
   <widget class="QCheckBox" name="xiu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位1 </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>22</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>48</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位11</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_12">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位12</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_13">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位13</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_14">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位14</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_15">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位15</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="xiu_16">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>48</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>相位16</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>311</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>尾部绿灯</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>306</x>
     <y>311</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>尾部红灯</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>546</x>
     <y>311</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>尾部黄灯</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxg">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>311</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxr">
   <property name="geometry">
    <rect>
     <x>416</x>
     <y>311</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxy">
   <property name="geometry">
    <rect>
     <x>646</x>
     <y>311</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="display">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxnumber">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>17</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
  </widget>
  <widget class="QPushButton" name="butfind">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>查 询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butset">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>修 改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butquery">
   <property name="geometry">
    <rect>
     <x>633</x>
     <y>13</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询
已设</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>26</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>已设行数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butclearall">
   <property name="geometry">
    <rect>
     <x>693</x>
     <y>13</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空
全表</string>
   </property>
  </widget>
  <widget class="QLabel" name="hang">
   <property name="geometry">
    <rect>
     <x>569</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butcleaer">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>清空行</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
